### DISCUSSION TITLE
Have a Feature Request?

### DISCUSSION THREAD

**Discussion by boylett** [Sheezy-Art/sheezy-hub#2] [Feature Requests]
In this early development stage, we aren't accepting feature requests. However, you're more than welcome to discuss the site's functionality in our [Discord server](https://discord.gg/3fKDMFNM3h).

**Comment by CrescentCaribou** [comment]
I'd like to request that [Kitsu](https://kitsu.io/explore/anime), [CharacterHub](https://characterhub.com/home), & [Pronouns.Page](https://en.pronouns.page/@CrescentCaribou) get social link icons!

**Reply by xnyoom** [reply]
These have been added! Thanks for the suggestion.
